pyweek36/main.py

```diff
@@ -22,19 +21,23 @@ def __init__(self, width, height, title):
 
         super().__init__(width, height, title)
 
-        self.next_spread = None
-        self.player_sprite: PlayerSprite = PlayerSprite()
+        self.player_sprite: PlayerSprite = PlayerSprite(self)
```

Comment:
I think everything in the __init__ should only assign to None, and assignments are done in setup

Reply:
I think initializing `SpriteList`s in `__init__` is fine (since afaik it's just a list with a few extra features), but yeah, creating a player should probably be done in `setup`